catalog/core/catalog-core-impl/filter-proxy/src/test/java/ddf/catalog/filter/proxy/adapter/test/FilterAdapterTest.java

```diff
@@ -95,25 +96,25 @@
             new DefaultPeriodDuration(DAY_IN_MILLISECONDS);
 
     @Test
-    public void includeFilter() {
+    public void testIncludeFilter() {
```

Comment:
I figured that was the case but wanted to confirm.  Thanks for making it consistent.